File: app/src/main/java/com/pkfcooperparry/template/data/model/db/visit/Condition.java
package com.pkfcooperparry.template.data.model.db.visit;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import lombok.Getter;
import lombok.Setter;

public class Condition extends RealmObject {
    @Getter
    @Setter
    @SerializedName("id")
    @Expose
    @PrimaryKey
    private Long id;
    @Getter
    @Setter
    @SerializedName("elementId")
    @Expose
    private Long elementId;
    @Getter
    @Setter
    @SerializedName("type")
    @Expose
    private String type;
    @Getter
    @Setter
    @SerializedName("value")
    @Expose
    private Value value;

    public Condition() {
    }
}

File: app/src/main/java/com/televantou/template/data/model/db/visit/Condition.kt
package com.televantou.template.data.model.db.visit

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Condition : RealmObject() {
    @SerializedName("id")
    @Expose
    @PrimaryKey
    var id: Long? = null
    @SerializedName("elementId")
    @Expose
    var elementId: Long? = null
    @SerializedName("type")
    @Expose
    var type: String? = null
    @SerializedName("value")
    @Expose
    var value: Value? = null
}
